tests/main/disk-space-awareness/task.yaml

```diff
@@ -49,7 +50,7 @@ execute: |
 
   # note, installing hello-world also pulls core snap.
   echo "And succeeds with enough space"
-  resize 150M
+  resize "$SUFFICIENT_SIZE"
   snap install hello-world
```

Comment:
was the test previously failing on this step? it could perhaps be due to the fact that we install the snapd snap by default right now too, in which case the comment above may need to be updated

Reply:
So installing `hello-world` pulls both `core` and `snapd` now? I thought it would continue to just pull `core`.

Reply:
On classic systems, it should since: https://github.com/canonical/snapd/pull/14173

Reply:
Isn't snapd already installed as well since it's what's being tested by spread? Or is it pulling in latest/stable snapd from the store in addition to the testing snapd?

Reply:
No idea, you’d have to get in the spread test to see.

Reply:
> Isn't snapd already installed as well since it's what's being tested by spread? Or is it pulling in latest/stable snapd from the store in addition to the testing snapd?

Looks like the test may be 'hiding' snaps installed in prepare by mounting tmpfs over /var/lib/snapd, so it's as if we're starting from scratch.